workflow/workflow.go

```diff
@@ -197,6 +197,11 @@ func GetMetricsHandler(ctx Context) metrics.Handler {
 	return internal.GetMetricsHandler(ctx)
 }
 
+// GetUnhandledSignalNames returns signal names that have not been consumed.
```

Comment:
But they may have been consumed, just not consumed completely. More accurately:

```suggestion
// GetUnhandledSignalNames returns signal names that have unconsumed signals.
```